app/controllers/api/v2/results_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    class ResultsController < BaseController
+      before_action :load_team, :load_project, :load_experiment, :load_task
+      before_action only: %i(show update destroy) do
+        load_result(:id)
+      end
+      before_action :check_create_permissions, only: :create
+      before_action :check_delete_permissions, only: :destroy
+      before_action :check_update_permissions, only: :update
+
+      def index
+        results = timestamps_filter(@task.results).page(params.dig(:page, :number))
+                                                  .per(params.dig(:page, :size))
+        render jsonapi: results, each_serializer: ResultSerializer,
+               include: (%i(text table file) << include_params).flatten.compact
```

Comment:
Don't include elements by default, let's keep it inline with steps implementation